### PR TITLE
Feature/video

### PR DESCRIPTION
**Pull request by bartosz-lipinski**

### CODE REVIEW

File: js/packages/common/src/actions/metadata.ts [hunk 1]
```diff
@@ -61,6 +66,8 @@ export interface IMetadataExtension {
   description: string;
   // preview image absolute URI
   image: string;
+  animation_url?: string;
```
Comment (bhgames) [inline js/packages/common/src/actions/metadata.ts:69]: more snake case?

File: js/packages/web/src/components/ArtContent/index.tsx [hunk 2]
```diff
@@ -116,15 +119,16 @@ const VideoArtContent = ({
       <video
         className={className}
         playsInline={true}
-        autoPlay={false}
+        autoPlay={true}
```
Comment (bhgames) [inline js/packages/web/src/components/ArtContent/index.tsx:122]: you sure?

File: js/packages/web/src/views/art/index.tsx [hunk 3]
```diff
@@ -79,6 +87,31 @@ export const ArtView = () => {
...
                       </span>
+                      {/* <Button
+                        onClick={async () => {
+                          if(!art.mint) {
+                            return;
+                          }
+                          const mint = new PublicKey(art.mint);
+
+                          const account = accountByMint.get(art.mint);
+                          if(!account) {
```
Comment (bhgames) [inline js/packages/web/src/views/art/index.tsx:98]: This looks interesting...why keep it commented out

File: js/packages/web/src/components/ArtContent/index.tsx (hunk 2 above)
Comment (bartosz-lipinski) [reply js/packages/web/src/components/ArtContent/index.tsx:127]: yes

File: js/packages/web/src/views/art/index.tsx (hunk 3 above)
Comment (bartosz-lipinski) [reply js/packages/web/src/views/art/index.tsx:99]: little secret

**Pull request merged**